global_config/register_domain.sh

```diff
@@ -0,0 +1,10 @@
+CONFIG=production
+CONFIG_SHORT=pd
+AZ_SUBSCRIPTION=s189-teacher-services-cloud-production
+RESOURCE_PREFIX=s189p01
+ENV_TAG=Prod
+DNS_ZONE=rtt
+RESOURCE_GROUP_NAME=${RESOURCE_PREFIX}-${DNS_ZONE}domains-rg
+KEYVAULT_NAME=${RESOURCE_PREFIX}-${DNS_ZONE}domains-kv
+STORAGE_ACCOUNT_NAME=${RESOURCE_PREFIX}${DNS_ZONE}domainstf
```

Comment:
Change `RESOURCE_PREFIX` to `RESOURCE_NAME_PREFIX`

Reply:
done